environments/workspaces-as-code/index.md

```diff
@@ -23,6 +24,26 @@ how to configure environments and their supporting infrastructure.
 You can find a fully populated workspace template and descriptions of each field
 in our [syntax guide](templates.md).
 
+## Creating a workspace using a template
+
+To create a new workspace using a template, go to **New Workspace** > **Create
+from Template**.
+
+![Create from template button](../../assets/create-from-template.png)
+
+When prompted, provide:
+
+- **Workspace Name**: A friendly name for your workspace
```

Comment:
Suggestion: we don't mention `friendly` as it's corresponding to different IDs/representations at the data layer, which do not seem as beneficial in this context

Reply:
I stole that language from the current environment form:
https://codercom-g9vx92axl-codercom.vercel.app/docs/environments/getting-started#2-create-an-environment